### ISSUE TITLE
Make the JacksonCollege Repos private.

### ISSUE THREAD

**Issue by BigActual** [JacksonCollege/jc-theme-2015#283]
@banksashleym - How about it? Can you make these private? Check out the options for GitHub ownership under your account profile in the top left corner.

**Comment by banksashleym** [comment]
It looks like we have the free version of Github and it's saying that in order to make the repository private I will need to upgrade. I will check with Dotty to see if we can do this.

**Comment by BigActual** [comment]
Yes, very few of our clients, especially ones of your size run with a free repo for very long. Privacy and security, etc.

**Comment by d4mation** [comment]
@banksashleym 

Just a heads up, Private Repos are no longer a paid-only feature for Organization accounts. So this is just a switch that can be flipped.

**Comment by banksashleym** [comment]
That would be awesome! I would prefer it to be private, thank you!
